specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2019-10-01/componentFeaturesAndPricing_API.json

```diff
@@ -0,0 +1,484 @@
+        "operationId": "ComponentCurrentBillingFeatures_Get",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "An Application Insights component billing features definition.",
+            "schema": {
+              "$ref": "#/definitions/ApplicationInsightsComponentBillingFeatures"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ComponentCurrentBillingFeaturesGet": {
+            "$ref": "./examples/CurrentBillingFeaturesGet.json"
+          }
+        }
+      },
+      "put": {
+        "description": "Update current billing features for an Application Insights component.",
+        "operationId": "ComponentCurrentBillingFeatures_Update",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          },
+          {
+            "name": "BillingFeaturesProperties",
+            "description": "Properties that need to be specified to update billing features for an Application Insights component.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ApplicationInsightsComponentBillingFeatures"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful request when updating billing features for an Application Insights component. The updated current billing features are returned.",
+            "schema": {
+              "$ref": "#/definitions/ApplicationInsightsComponentBillingFeatures"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ComponentCurrentBillingFeaturesUpdate": {
+            "$ref": "./examples/CurrentBillingFeaturesUpdate.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Insights/components/{resourceName}/quotastatus": {
```

Comment:
This should follow the [common usage contract. ](https://armwiki.azurewebsites.net/api_contracts/UsagesAPIContract.html)